server/views/applications/pages/basic-information/reason-for-short-notice.njk

```diff
@@ -23,7 +23,7 @@
         fieldset: {
           legend: {
             text: page.question,
-            classes: "govuk-fieldset__legend--m"
+            classes: "govuk-fieldset__legend--l"
```

Comment:
I think we only want this to be large when there isn't a page title (i.e. for short notice applications but not emergency applications if the ticket is correct), unless the emergency applications title also needs removing (in which case the ticket needs revising)? Could add a little ternary logic here to check if there's a page title and set it to `l` or `m` accordingly?